cli/medperf/entities/cube.py

```diff
@@ -355,9 +363,9 @@ def run(
         cmd += " -Pplatform.accelerator_count=0"
 
         logging.info(f"Running MLCube command: {cmd}")
-        proc = pexpect.spawn(cmd, timeout=timeout)
-        proc_out = combine_proc_sp_text(proc)
-        proc.close()
+        with pexpect.spawn(cmd, timeout=timeout) as proc:
+            proc_out = combine_proc_sp_text(proc)
+
         if output_logs is None:
```

Comment:
This doesn't seem to do the trick
pressing CTRL+C is not stopping the child process

Reply:
I see you are totally right.
When `medperf` -> `mlcube` -> `container` stack is active, stopping (ctrl+C) medperf stops `mlcube`, but does not stop container. Though when mlcube is running directly, stopping mlcube stops container also. 
I'll debug it deeper...

Reply:
A separate issue for solving this: https://github.com/mlcommons/medperf/issues/531